cypress/e2e/signIn.cy.js

```diff
@@ -28,7 +28,31 @@ describe('Sign In page', () => {
     homePage.assertHeaderContainUsername(user.username);
   });
 
-  it('should not provide an ability to log in with wrong credentials', () => {
+  it('should not provide an ability to log in with wrong email', () => {
+    signInPage.visit();
+    cy.register(user.email, user.username, user.password);
+
+    const userEmail = '77' + user.email;
+    signInPage.typeEmail(userEmail);
+    signInPage.typePassword(user.password);
+    signInPage.clickSignInBtn();
+
+    cy.contains('Login failed!');
```

Comment:
Consider using data-qa attributes for selecting elements instead of relying on the text content of the elements. This makes your tests less brittle to changes in copy.

Reply:
Similar to the above